test/local/senders/email.js

```diff
@@ -779,6 +779,26 @@ describe(
       }
     )
 
+    it(
+      'sends request to fxa-email-server when the email pattern is right',
+      function() {
+        var message = {
+          email: 'emailservice.foo@restmail.net',
+          flowId: 'wibble',
+          subject: 'subject',
+          template: 'verifyLoginEmail',
+          uid: 'foo'
+        }
+
+        var stub = sinon.stub(mailer.emailService, 'sendMail')
+        mailer.send(message)
+          .then(
+            response => assert(stub.called),
+            err => assert.notOk
```

Comment:
No biggie, but fwiw we could also just delete this assertion and return the promise from `mailer.send` instead. That should also print a more useful error message for debugging purposes if the test ever starts failing.